packages/three-instanced-uniforms-mesh/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "three-instanced-uniforms-mesh",
-  "version": "0.44.0",
+  "version": "0.44.1",
```

Comment:
Please revert this; version bumps are handled by the publishing scripts.

Reply:
done